docs/source/kapsel/index.rst

```diff
@@ -182,7 +182,7 @@ The command line session will look something like::
 
 If you look at ``kapsel.yml`` now, you'll see bokeh and pandas
 listed under the ``packages:`` section. You will also see files
-such as ``envs/default/bin/bokeh`` since the packages have now
+such as ``envs/default/conda-meta/bokeh`` since the packages have now
```

Comment:
I don't think this is right. I have conda-meta/bokeh-0.12.0-py27_0.json, not conda-meta/bokeh.

I suspect the issue is that on Windows Aamna had something like Scripts\bokeh.exe, not bin/bokeh.
I'm having trouble coming up with a cross-platform file to mention, so maybe we should just say "you will also see Bokeh-related files in envs/default since the packages have now been installed" or maybe we should delete this sentence...